### PR TITLE
Implement the ability to delete a resume (#55)

### PR DESCRIPTION
**Pull request by cmsparks**
Implements #55 

Backend and frontend changes are present here. A new API endpoint was added as well (DELETE /api/resume)

### CODE REVIEW

File: controller/resume/impl.go [hunk 1]
```diff
@@ -92,3 +92,29 @@ func (controller *ResumeController) ApproveResume(ctx *context.Context) error {
...
+			err,
+		)
+	}
+
+	err = controller.svc.Resume.DeleteResume(req.Username)
+	if err != nil {
+		return ctx.JSONError(
+			http.StatusBadRequest,
+			"Failed Resumes Approval",
+			"could not mark resume as approved",
```
Comment (apappu9) [inline controller/resume/impl.go:114]: Error messages need to be updated, maybe something like this?
```suggestion
			"Failed Resumes Deletion",
			"could not delete resume",
```

File: service/resume/impl.go [hunk 2]
```diff
@@ -65,6 +65,15 @@ func (service *resumeImpl) ApproveResume(username string) error {
 	return nil
 }
 
+func (service *resumeImpl) DeleteResume(username string) error {
+	err := service.deleteResume(username)
+	if err != nil {
+		return fmt.Errorf("failed to approve resume: %w", err)
```
Comment (apappu9) [inline service/resume/impl.go:71]: same thing here
```suggestion
		return fmt.Errorf("failed to delete resume: %w", err)
```

Comment (cmsparks) [reply service/resume/impl.go:71]: Yup, let me go through and fix all instances of this.

Comment (cmsparks) [reply service/resume/impl.go]: I think those were the only two, should be good now.

**Pull request merged**